oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LucenePropertyIndex.java

```diff
@@ -1545,6 +1545,8 @@ private static Query tokenToQuery(String text, String fieldName, PlanResult pr,
             Query subQuery = new BooleanQuery();
             if (pr.indexDefinition.isDynamicBoostLiteEnabled()) {
                 subQuery = tokenToQuery(text, FieldNames.SIMILARITY_TAGS, analyzer);
+                // De-boosting dynamic boost based query so other clauses will have more relevance
+                subQuery.setBoost(0.001f);
```

Comment:
This used to be 0.0001f I think. [0]

[0] - https://git.corp.adobe.com/CQ/dam/blob/a4173e7086e399a40f0985df252bf6a80bc29fad/bundles/similarity-search/src/main/java/com/day/cq/dam/similaritysearch/internal/impl/ManageAutoTagsServiceImpl.java#L534-L535